core/src/main/java/hudson/ProxyConfiguration.java

```diff
@@ -540,6 +540,34 @@ public FormValidation doCheckPort(@QueryParameter String value) {
             return FormValidation.ok();
         }
 
+        /**
+         * Do check if the provided value is empty or composed of whitespaces.
+         * If so, return a validation warning.
+         *
+         * @param value the value to test
+         * @return a validation warning iff the provided value is empty or composed of whitespaces.
+         */
+        private static FormValidation doCheckProxyCredentials(String value) {
```

Comment:
💡 I would recommend you to avoid naming method "doXxx" if they are not intended to be used as Stapler endpoint.